index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="css/index.css" />
+		<script
+			type="module"
+			src="javascript/index.js"
+			crossorigin="anonymous"
+		></script>
```

Comment:
Add a js loading attribute like differ to make sure that the script gets executed after the DOM rendering.